Trakt/Model/TraktUser.cs

```diff
@@ -25,14 +32,18 @@ public class TraktUser
 
         public bool Scrobble { get; set; }
 
-        public IReadOnlyList<string> LocationsExcluded { get; set; }
+        public string[] LocationsExcluded { get; set; }
```

Comment:
Why?

Reply:
Reverting the change made in #28 because it introduces this exception:

System.InvalidOperationException: Cannot serialize member 'Trakt.Model.TraktUser.LocationsExcluded' of type 'System.Collections.Generic.IReadOnlyList